File: CS/WebSite/clsDetails.cs
using System;
using System.Data;
using System.Configuration;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using System.Collections.Generic;

public class clsDetails : List<clsDetail>
{
    public clsDetails()
    {
        this.Add(new clsDetail("A", "M", "Mr", "Address1"));
        this.Add(new clsDetail("B", "M", "Dr", "Address2"));
        this.Add(new clsDetail("C", "M", "Mr", "Address3"));
        this.Add(new clsDetail("D", "M", "Mr", "Address4"));
        this.Add(new clsDetail("E", "F", "Mrs","Address5"));
        this.Add(new clsDetail("F", "M", "Dr","Address6"));
        this.Add(new clsDetail("G", "M", "Mr","Address7"));
        this.Add(new clsDetail("H", "F", "Ms","Address8"));
        this.Add(new clsDetail("I", "F", "Dr","Address9"));
        this.Add(new clsDetail("J", "M", "Mr","Address10"));
        this.Add(new clsDetail("K", "M", "Mr", "Address11"));
    }

    public clsDetail GetDetails(string code)
    {
        clsDetail det = new clsDetail("undefined","undefined","undefined","undefined");
        for (int i = 0; i < this.Count; i++)
        {
            string currentCode = this[i].Code;
            if (currentCode == code)
            {
                det =  this[i];
                break;
            }
           
        }
        return det;
    }
}

File: VB/WebSite/clsDetails.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls
Imports System.Collections.Generic

Public Class clsDetails
	Inherits List(Of clsDetail)
	Public Sub New()
		Me.Add(New clsDetail("A", "M", "Mr", "Address1"))
		Me.Add(New clsDetail("B", "M", "Dr", "Address2"))
		Me.Add(New clsDetail("C", "M", "Mr", "Address3"))
		Me.Add(New clsDetail("D", "M", "Mr", "Address4"))
		Me.Add(New clsDetail("E", "F", "Mrs","Address5"))
		Me.Add(New clsDetail("F", "M", "Dr","Address6"))
		Me.Add(New clsDetail("G", "M", "Mr","Address7"))
		Me.Add(New clsDetail("H", "F", "Ms","Address8"))
		Me.Add(New clsDetail("I", "F", "Dr","Address9"))
		Me.Add(New clsDetail("J", "M", "Mr","Address10"))
		Me.Add(New clsDetail("K", "M", "Mr", "Address11"))
	End Sub

	Public Function GetDetails(ByVal code As String) As clsDetail
		Dim det As New clsDetail("undefined","undefined","undefined","undefined")
		For i As Integer = 0 To Me.Count - 1
			Dim currentCode As String = Me(i).Code
			If currentCode = code Then
				det = Me(i)
				Exit For
			End If

		Next i
		Return det
	End Function
End Class